src/llama_stack_provider_kft/config.py

```diff
@@ -25,6 +25,7 @@ class InstructLabKubeFlowPostTrainingConfig(BaseModel):
     job_timeout: int = 86400
     nnodes: int = 2
     delete_after_done: bool = False
+    keep_last_checkpoint_only: bool = False
```

Comment:
I'd be tempted to suggest it defaults to True.

Reply:
This will blow phase 2, so I'd prefer enabling this on purpose for phase 1 only. What do you think?

Reply:
Ah I missed that, whatever works best is fine to me.